core/tests/webdriverio_utils/forms.js

```diff
@@ -757,7 +765,7 @@ var CodeStringEditor = function(elem) {
     setValue: async function(code) {
       var stringEditorTextArea = await elem.$('textarea');
       await action.clear('String Editor Text Area', stringEditorTextArea);
-      await action.setValue(
+      await action.addValue(
         'String Editor Text Area',
         stringEditorTextArea,
         code);
```

Comment:
Ditto why addValue? I am asking this as you have used setValue earlier.

Reply:
Same as here https://github.com/oppia/oppia/pull/15969/files#r970561820